subworkflows/nf-core/variantcalling/manta/somatic/main.nf

```diff
@@ -101,15 +144,27 @@ workflow RUN_MANTA_SOMATIC {
         MERGE_MANTA_SMALL_INDELS.out.vcf,
         manta_candidate_small_indels_vcf.no_intervals
     ).map{ meta, vcf ->
-        [[patient:meta.patient, normal_id:meta.normal_id, tumor_id:meta.tumor_id, sex:meta.sex, id:meta.tumor_id + "_vs_" + meta.normal_id],
+        [[
+            id:         meta.tumor_id + "_vs_" + meta.normal_id,
+            normal_id:  meta.normal_id,
+            patient:    meta.patient,
+            sex:        meta.sex,
+            tumor_id:   meta.tumor_id,
```

Comment:
Does the `variantcaller:"manta"` have to be added here as well?

Reply:
no these vcfs don't get annotated, strelka just needs them for bp computation. the meta field variantcaller is just used for writing the annotated vcfs to the proper output folders.